QUEST_LV_0100.tsv

```diff
@@ -4096,108 +4096,108 @@ QUEST_LV_0100_20150317_004095	Polyna states that the stolen supplies must be ret
-QUEST_LV_0100_20150317_004174	Talk to the Worrying Owl Statue.
-QUEST_LV_0100_20150317_004175	Do as the Old Owl Statue says and bring the fragment of memory to the Worrying Owl Statue in Bandyma Crossroad.
-QUEST_LV_0100_20150317_004176	Awaken the memory of the sleeping Owl Statue
-QUEST_LV_0100_20150317_004177	In order to defeat Gaigalas, you need power of the Owl Statues that fell asleep without completing the magic. Awaken the memory of sleeping owl statues and complete the magic that will weaken Gaigalas.
-QUEST_LV_0100_20150317_004178	Report to the Worrying Owl Statue
-QUEST_LV_0100_20150317_004179	Completed the magic that will weaken Gaigalas. Return and tell it to the Worrying Owl Statue.
-QUEST_LV_0100_20150317_004180	Summon and defeat Gaigalas
-QUEST_LV_0100_20150317_004181	Completed the magic that will weaken Gaigalas. Summon Gaigalas and defeat it.
-QUEST_LV_0100_20150317_004182	Defeat Gaigalas in Ishimpyti Entrance
-QUEST_LV_0100_20150317_004183	Completed the magic that will weaken Gaigalas. Defeat Gaigalas in Ishimpyti Entrance.
-QUEST_LV_0100_20150317_004184	Talk to Archaeologist Dezik
-QUEST_LV_0100_20150317_004185	Archaeologist Dezik looks anxious. Talk to him.
-QUEST_LV_0100_20150317_004186	Defeat the nearby monsters for Archaeologist Dezik
-QUEST_LV_0100_20150317_004187	Dezik asked for your help in getting rid of the monsters nearby so that he can go look for his assistant Irine.
-QUEST_LV_0100_20150317_004188	This will be safe enough for Dezik to go to the soldiers. Return to Dezik.
-QUEST_LV_0100_20150317_004189	Talk to Technician Heinen
-QUEST_LV_0100_20150317_004190	{memo X}You need to use the cable car to go over to the Radas Fork Road easily. Talk to Technician heinen to use the cable car.
-QUEST_LV_0100_20150317_004191	Defeat the monsters around the cable car
+QUEST_LV_0100_20150317_004099	$Talk to Officer Luders
+QUEST_LV_0100_20150317_004100	$Ready to hunt Sparnas. Talk to Officer Luders.
+QUEST_LV_0100_20150317_004101	$Lure Sparnas in the last quarters. 
+QUEST_LV_0100_20150317_004102	$Officer Luders told you to lure Sparnas with bait and defeat it. Place the bait in Malkos Felled Area and defeat Sparnas when it appears. 
+QUEST_LV_0100_20150317_004103	$Report to Officer Luders
+QUEST_LV_0100_20150317_004104	$Successfully defeated Sparnas. Report back to Officer Liudas. 
+QUEST_LV_0100_20150317_004105	$Defeat the lured Sparnas. 
+QUEST_LV_0100_20150317_004106	$Talk to Liaison Officer Niels
+QUEST_LV_0100_20150317_004107	$Arrived safely at the Gateway of the Great King. Find the person in charge. 
+QUEST_LV_0100_20150317_004108	$Find the missing archivist, Jonas.
+QUEST_LV_0100_20150317_004109	$Liaison Officer Niels is looking for an old man named Jonas. Be sure to bring him back when you find him.
+QUEST_LV_0100_20150317_004110	$Find Archivist Jonas
+QUEST_LV_0100_20150317_004111	$Talk to Jonas
+QUEST_LV_0100_20150317_004112	$You were proved as the right person by the Eye of the Great King. Talk to Jonas.
```

Comment:
"You were proved as the right person by the Eye of the Great King. Talk to Jonas." -> "You were **proven to be** the right person by the Eye of the Great King. Talk to Jonas."